src/autopas/selectors/TraversalSelector.h

```diff
@@ -38,21 +39,9 @@ namespace autopas {
  */
 template <class ParticleCell>
 class TraversalSelector {
- public:
-  /**
-   * Dummy constructor such that this class can be used in maps
-   */
-  TraversalSelector() : _dims({0, 0, 0}), _cutoff(1.0), _cellLength({0.0, 0.0, 0.0}) {}
-  /**
-   * Constructor of the TraversalSelector class.
-   * @param dims Array with the dimension lengths of the domain
-   * @param cutoff Cutoff radius
-   * @param cellLength cell length.
-   */
-  TraversalSelector(const std::array<unsigned long, 3> &dims, const double cutoff = 1.0,
-                    const std::array<double, 3> &cellLength = {1.0, 1.0, 1.0})
-      : _dims(dims), _cutoff(cutoff), _cellLength(cellLength) {}
+  friend TraversalSelectorInfo<ParticleCell>;
```

Comment:
TraversalSelectorInfo params public. We dont linke friends.
-> remove underscores